web/src/pages/register.tsx

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import { useState, ChangeEvent } from 'react';
+import { MdArrowBackIosNew } from 'react-icons/md';
+import Button from '../components/Inputs/Button';
+import LinkIconButton from '../components/Inputs/LinkIconButton';
+import TextField from '../components/Inputs/TextField';
+
+
+export default function RegisterPage() {
+
+    const [state, setState] = useState({
+        email:"",
+        username: "",
+        password: "",
+        retype: ""
+    });
+
+    const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+        setState(prev => ({...prev, [e.target.name]:e.target.value}));
+    }
+
+
+    return (
+        <div className={"h-screen"}>
+            <div className={"mb-12"}>
+                <nav className={"sticky top-0 mb-16 h-12 rounded-b-xl px-4"}>
```

Comment:
At this point we should remove the back navigation altogether. I don't really see why we need this